src/Editor/GUI/Editors/BoxShapeEditor.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "IShapeEditor.hpp"
+
+namespace Component {
+    class Shape;
+}
+
+namespace GUI {
+    /// Box shape editor for physics components.
+    class BoxShapeEditor : public IShapeEditor {
+        public:
+            /// Constructor
+            BoxShapeEditor() = default;
+
+            /// Box type label.
+            /**
+             * @return "Box" string.
+             */
+            virtual const char* Label() const override { return "Box"; }
```

Comment:
`virtual` is redundant due to `override`

Reply:
Fixed. Removed the other occurences in the same file and also in the plane- and sphere shape editors.